src/test/kotlin/io/sip3/commons/domain/payload/RtpReportPayloadTest.kt

```diff
@@ -0,0 +1,199 @@
+/*
+ * Copyright 2018-2019 SIP3.IO, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.sip3.commons.domain.payload
+
+import io.sip3.commons.util.remainingCapacity
+import org.junit.jupiter.api.Assertions.*
+import org.junit.jupiter.api.Test
+import java.nio.charset.Charset
+
+class RtpReportPayloadTest {
+
+    companion object {
+
+        const val CALL_ID = "f81d4fae-7dec-11d0-a765-00a0c91e6bf6@foo.bar.com"
+        const val CALL_ID_VALUE_LENGTH = CALL_ID.length + 3
+
+        private fun getRtpReportPayload(callId: String?): RtpReportPayload {
```

Comment:
Do we really have a case when `callId` here is undefined?

Reply:
For some reasons, we may have no SDP information from Salto(e.g. Captain restarted and RTP session is on air).